release-notes/major9/09.23.0/valtimo-backend-libraries.md

```diff
@@ -22,6 +22,26 @@ The following features were added:
    }
   ```
 
+* **A document can now be assigned to a user**
+
+  There are two methods of assigning a user to a document.
+  
+  1) Using the endpoint `POST /api/document/{documentId}/assign` and providing the assignee id in the request body.
```

Comment:
I think this should go on a different page. The release notes target more than just developers, so try and keep the description here as functional as possible. And for more details, link to a corresponding using valtimo/extending valtimo page.